test/service/test_response.py

```diff
@@ -2552,9 +2636,13 @@ def test_matrix_cell_count(self):
                         s['fileType']: s['matrixCellCount']
                         for s in hit['fileTypeSummaries']
                     }
-                    self.assertEqual(expected_counts, actual_counts)
+                    self.assertEqual(expected, actual_counts)
 
         # Verify matrix_cell_counts across all hits in the 'files' endpoint
+        expected = {
+            'Rds': 54140 * 3,  # 3 analysis files
+            'fastq.gz': 0  # 360 sequence files
```

Comment:
Not necessarily a review comment, but I'm curious how there can be 360 sequence files and no cells.

Reply:
`matrix_cell_count` is a field that only analysis files have.

https://github.com/DataBiosphere/hca-metadata-api/blob/f80eddd486bfd59e07f0619cf156fb9c0efbf6ab/src/humancellatlas/data/metadata/api.py#L828
https://schema.humancellatlas.org/type/file/6.3.0/analysis_file

With that said the 360 number is incorrect. While the canned bundle does have 360 sequence files in it, only 24 of those have `"is_stitched": false` resulting in a total of 27 hits from the `/index/files` endpoint.